lib/dangermattic/plugins/ios_misc_checks.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'yaml'
+
+module Danger
+  PODFILE_LOCK = 'Podfile.lock'
+  PODFILE_LOCK_DEPENDENCIES_ENTRY = 'DEPENDENCIES'
+  DEFAULT_PODFILE_LOCK_PATH = './Podfile.lock'
+
+  # Plugin for miscellaneous iOS checks.
+  class IosMiscChecks < Plugin
```

Comment:
Same feedback as above, I noted your paragraph in the PR description but I still think we should use a better name, like `PodfileChecks`?